dgraph/cmd/zero/http.go

```diff
@@ -231,9 +234,44 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) zeroHealth(ctx context.Context) (*api.Response, error) {
+	if ctx.Err() != nil {
+		return nil, ctx.Err()
+	}
+
+	healthAll := []pb.HealthInfo{pb.HealthInfo{
```

Comment:
why is this a list?

Reply:
No need for this to be a list, corrected.